fixtures/multiplatform_µFeature_unit_tests_with_explicit_dependencies/Modules/ModuleA/Tests/Sources/FrameworkATests.swift

```diff
@@ -0,0 +1,23 @@
+import Mocker
+import ModuleATestSupporting
+import XCTest
+
+class FrameworkATests: XCTestCase {
+    var sut: URLSession!
+
+    override func setUp() {
+        super.setUp()
+
+        let configuration = URLSessionConfiguration.default
+        configuration.protocolClasses = [MockingURLProtocol.self]
+
+        sut = URLSession(configuration: configuration)
+
+        NetworkResponseMocks.testMock.register()
```

Comment:
Acceptance tests have a higher maintenance cost than unit tests; therefore, we should aim to keep it as low as possible. With that in mind, I'd consider removing `Mocker` from this acceptance test unless you believe it's core to the scenario and we should keep it. An external dependency like this in a test is something we might need to get familiar to understand the test and to keep up to date.

Reply:
What I wanted to test with this is a dependency that has a dependency to XCTest in a unit test test target. Do you think it would be beneficial to "Mock" an external dependency by a local one which simply depends on XCTest ? Then we would not rely on an external one here